apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,88 @@
+
+3. Corrupt linker path of shared library on macOS due to specification `--destdir` in `meson install` step - We were facing this issue when we used `spin` to build SciPy (after implementing the prototype in [gh-21674](https://github.com/scipy/scipy/pull/21674)). When we did, `spin test`, then it was unable to load, `libsf_error_state.dylib` on macOS. One cause for this issue is that `spin` sets `/usr` or `C:\` as default value for `--prefix`. It also specifies `--destdir` in `meson` install step. `--destdir` is meant for packaging, as an actual staging area, while for spin the final install directory is `build-install` (by default), and therefore there is no intent to later put this package into /usr`or`C:\`. Hence we removed `--destdir` usage and set `--prefix` to installation directory. Here's the pull request, [scientific-python/spin#257](https://github.com/scientific-python/spin/pull/257) which fixes this issue.
+
+`DEV: use `spin` (prototype)` [[gh-21674]](https://github.com/scipy/scipy/pull/21674) is close to merging. We are waiting for [scientific-python/spin#257](https://github.com/scientific-python/spin/pull/257) to be merged. Once in, we are ready to start using `spin` in SciPy. The SciPy CI is using `spin` in the above PR. So our work is completely tested. Here's the current status of how `spin` usage looks like,
+
+**SciPy build experience with `spin` and `dev.py`**
+
+**With `dev.py`**
+
+<script src="https://asciinema.org/a/RrtqCRlxIaA6pNS6BQchJyUmN.js" id="asciicast-RrtqCRlxIaA6pNS6BQchJyUmN" async="true"></script>
+
+**With `spin`**
+
+<script src="https://asciinema.org/a/FJzWyQun71g796n5kFI1l6dxv.js" id="asciicast-FJzWyQun71g796n5kFI1l6dxv" async="true"></script>
+
+As you can see there is no difference between look and feel of `python dev.py build` and `spin build`. It works exactly the same. However, the issues which were there with `dev.py` are not observed with `spin`. Take a look at the following videos,
+
+**`spin` keeps pdb command history intact while `dev.py` interferes with it [[gh-16452]](https://github.com/scipy/scipy/issues/16452)**
+
+**With `dev.py`**
+
+<script src="https://asciinema.org/a/shvsE0PfbGMsviuthoKfIQlI6.js" id="asciicast-shvsE0PfbGMsviuthoKfIQlI6" async="true"></script>
+
+**With `spin`**
+
+<script src="https://asciinema.org/a/wcwYTiPQzey2uYt1082w63Rgz.js" id="asciicast-wcwYTiPQzey2uYt1082w63Rgz" async="true"></script>
+
+As you can see `spin` usage doesn't interfere with PDB's history. This issue was there with `dev.py`.
+
+**Using homebrew installed Python to build SciPy [[gh-18998]](https://github.com/scipy/scipy/issues/18998) - `dev.py` vs `spin`**
+
+<script src="https://asciinema.org/a/aZstdaf6B6nlkW9JFccT7z7Dr.js" id="asciicast-aZstdaf6B6nlkW9JFccT7z7Dr" async="true"></script>
+
+The above video showcases how `spin` works fine with Homebrew installed Python 3.10. The build completes. However, `dev.py` fails because it is unable to resolve paths correctly.
```

Comment:
It would be good to add acknowledgements - this was done with NASA funding. Can you please copy the phrasing at the end of Albert's recent blog post for this?